LayoutTests/fast/css/css-text-unknown-element.html

```diff
@@ -0,0 +1,13 @@
+<script>
+function jsfuzzer() {
+	if(window.testRunner)
+        testRunner.dumpAsText();
+	unknownElement = document.createElement("unknown");
+	try {  elementStyle = unknownElement.style; } catch(e) { }
+	try { elementStyle["border"] = "menu"; } catch(e) { }
+	try { cssText = elementStyle.cssText; } catch(e) { }
+}
+</script>
+<body onload=jsfuzzer()>
+<p>PASS if no crash</p>
+</body>
```

Comment:
```
<script>
function test() {
    if (window.testRunner)
        testRunner.dumpAsText();
    const div = document.createElement("div");
    const elementStyle = div.style;
    elementStyle.border = "menu";
    cssText = elementStyle.cssText;
}
</script>
<body onload="test()">
<p>PASS if no crash</p>
```

nit: can you use 4 space indentation, and format the test properly? The snippet above should repro the crash.